### ISSUE TITLE
Кинопоиск отвалился =(

### ISSUE THREAD

**Issue by yakot93** [anvilabs/movie-api#1]
Не возможно воспользоваться Кинопоиском. Возвращает на любой запрос "[key:false]"

**Comment by yenbekbay** [comment]
@yakot93 Да, уже некоторое время не срабатывает дыра, которую я использовал для доступа к их API. Пока, к сожалению, другого способа не нашёл.

**Comment by yakot93** [comment]
@yenbekbay очень жаль..

**Comment by pushkarskiy** [comment]
Почему то некоторые запросы вываливаются в ошибку вида 

`error: TypeError: Assignment to constant variable.`

kinopoisk id для теста 1009145

**Comment by pushkarskiy** [comment]
Еще сейчас вижу в логах вот такую ошибку довольно часто `The header content contains invalid characters` nodejs 8 версии

**Comment by yenbekbay** [comment]
@yakot93 @pushkarskiy Кажется, получилось исправить. Если библиотека ещё нужна, можете проверить [последний релиз](https://github.com/anvilabs/movie-api/releases/tag/v0.14.6).
